### PR TITLE
[14.0][ADD] sale_commission_queued

### PR DESCRIPTION
**Pull request by ilyasProgrammer**

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional review ok

@ilyasProgrammer please add description

@pedrobaeza what do you think?

File: sale_commission/wizard/wizard_invoice.py [hunk 1]
```diff
@@ -41,6 +41,18 @@ def _default_from_settlement(self):
 
     def button_create(self):
         self.ensure_one()
+        settlements, invoices = self.create_settlements_and_invoices()
+        # go to results
+        if len(settlements):
```
Comment (pedrobaeza) [inline sale_commission/wizard/wizard_invoice.py]:
```suggestion
        if invoices:
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
An improvement may be to enqueue each invoice creation separately.

**Review by elvise — APPROVED** [review]
LGTM!

**Pull request merged**